flac.md

```diff
@@ -488,30 +488,59 @@ Value | Description
 
 The size of the samples stored in the subframe is the subframe sample size reduced by k bits. Decoded samples must be shifted left by k bits.
 
-## SUBFRAME_CONSTANT
-Data      | Description
-:---------|:-----------
-`u(n)`    | Unencoded constant value of the subblock, n = frame's bits-per-sample.
+## Constant subframe
+In a constant subframe only a single sample is stored. This sample is stored as a signed integer number, big-endian, signed two's complement. The number of bits used to store this sample depends on the bit depth of the current subframe. This bit depth of a subframe is equal to the bit depth of the corresponding subblock, minus the number of wasted bits in that subblock, plus 1 bit when it has been converted to a side-channel subframe. See also the [section on interchannel decorrelation](#interchannel-decorrelation) and the [section on wasted bits per sample flag](#wasted-bits-per-sample-flag).
 
-## SUBFRAME_FIXED
-Data       | Description
-:----------|:-----------
-`u(n)`     | Unencoded warm-up samples (n = frame's bits-per-sample \* predictor order).
-`RESIDUAL` | Encoded residual
+## Verbatim subframe
+A verbatim subframe stores all samples unencoded. See [section on Constant subframe](#constant-subframe) on how samples are stored unencoded. The number of samples that need to be stored in a subframe is given by the blocksize in the frame header.
```

Comment:
Suggestion: "A verbatim subframe stores `blocksize` samples unencoded in sequential order."
Suggestion: "on how a sample is stored unencoded"